README.md

```diff
@@ -28,7 +28,7 @@ For development purposes, you can set up `100eyes` using Docker. To perform the
 
     ```bash
     $ cd ~/100eyes
-    $ docker-compose build
+    $ docker compose build
```

Comment:
`docker-compose` is part of v1, which uses python, and has been deprecated. It has been migrated to Go in v2 and now is `docker compose`. See [docker docs](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command) for more info.

Reply:
Weird thing is that running `docker compose up SERVICE` starts all services, but `docker-compose up SERVICE` starts only the service you want to start. For example, you want to use your local db/rails server, but want to start the `mailserver`.... I checked the [docker compose up docs](https://docs.docker.com/engine/reference/commandline/compose_up) and it seems to indicate that it should be possible to start a single service, but it doesn't work for me :shrug: